include/miopen/miopen.h

```diff
@@ -346,12 +346,10 @@ MIOPEN_DECLARE_OBJECT(miopenReduceTensorDescriptor);
  */
 typedef enum
 {
-    miopenHalf  = 0, /*!< 16-bit floating point (Fully supported) */
-    miopenFloat = 1, /*!< 32-bit floating point (Fully supported) */
-    miopenInt32 = 2, /*!< 32-bit int point (Partially supported) */
-    miopenInt8  = 3, /*!< 8-bit int point (Partially supported) */
-    miopenInt8x4 =
```

Comment:
It would be useful to comment that this value is reserved and should not be used

Reply:
@averinevg Obsolete member is commented out like in `miopenReduceTensorOp_t` (while it seems questionable to me)